cli/signal_unix.go

```diff
@@ -7,8 +7,24 @@ import (
 	"syscall"
 )
 
+// StopSignals are used to gracefully exit.
+// An example is exiting provisioner daemons but not canceling
+// jobs, allowing a successful and clean exit.
+var StopSignals = []os.Signal{
+	syscall.SIGTERM,
+}
+
+// InterruptSignals are used to less gracefully exit.
+// An example is canceling a job, waiting for a timeout,
+// and then exiting.
 var InterruptSignals = []os.Signal{
+	// SIGINT
 	os.Interrupt,
-	syscall.SIGTERM,
 	syscall.SIGHUP,
 }
+
+// KillSignals will force exit.
+var KillSignals = []os.Signal{
```

Comment:
Do we need this? `os.Kill` can't be handled so I don't foresee the need for `KillSignals`.

Reply:
Good point